packages/portal/src/components/DS4CH/DS4CHLandingHero.vue

```diff
@@ -5,6 +5,7 @@
       :style="imageCSSVars"
       role="img"
       :aria-label="heroImage?.image?.description"
+      :alt="heroImage?.image?.description"
```

Comment:
Why was included here?

Reply:
The lack of an alt attribute on this element was flagged as an a11y violation in the e2e tests. Not sure why that hasn't been flagged before.

Reply:
I think it's because of the test data, probably before we cloned production into test, that hero didn't have an image at all so didn't expect an alt text for accessibility reasons.